client/src/build-view.js

```diff
@@ -45,7 +45,7 @@ class Build extends React.Component {
 
   render() {
     const { name } = this.props.match.params
-    const url = `http://${name}.localhost:8080`
+    const url = `http://${name}.${window.location.host}`
```

Comment:
Is there any way using the dynamic browser location could fail?
There's `meta/env.meta.js` that should contain a statically configured URL based on build-time environment variables.

Reply:
It's done this way in `container-card.js` too. The url in `env.meta.js` has the protocol too as part of it, would it be better then just to parse the hostname out of that?

Reply:
@quvide do you need @bqqbarbhg 's input?

Reply:
I suppose it's fine like that, you could add another meta constant that doesn't have the protocol though